plugins/woocommerce/client/blocks/tests/e2e/tests/product-collection/product-collection-errors.block_theme.spec.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { Page } from '@playwright/test';
+import { test, expect, Editor, wpCLI } from '@woocommerce/e2e-utils';
+
+test.describe( 'Product Page: error notices when adding out-of-stock products', () => {
+	test( 'displays error notice when attempting to add product beyond stock limit', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
+		const productName = 'A Managed Stock';
+
+		wpCLI(
```

Comment:
Heads-up that `wpCLI()` needs to be awaited:

```suggestion
		await wpCLI(
```